command/agent/agent.go

```diff
@@ -22,6 +22,13 @@ const (
 
 	// Path to save local agent checks
 	checksDir = "checks"
+
+	// errSocketFileExists is the human-friendly error message displayed when
+	// trying to bind a socket to an existing file.
+	errSocketFileExists = "A file exists at the requested socket path %q. " +
```

Comment:
Can we surround the path with quotes or a special character? If the path has spaces it it, it might be difficult to distinguish.